### PR TITLE
Update graphql to ^0.9.0 and add temporary name for EnumType in type …

### PR DESCRIPTION
**Pull request by klimentLambevski**
…mapper

### CODE REVIEW

File: src/typeMapper.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
 
   if (sequelizeType instanceof ENUM) {
     return new GraphQLEnumType({
+      name: 'tempEnumName', //grpahql 0.9 requires name for EnumType. Using tempEnumName until its replaced with field generated name
```
Comment (mickhansen) [inline src/typeMapper.js]: Will break if you have multiple enums in your schema, which would be most.

Comment (klimentLambevski) [inline src/typeMapper.js]: the name is replaced after type mapper is called so there is no problem for now, it is not the best solution  but maybe you can suggest better

**Comment by toxsick** [comment]
Hi guys,

is there any progress on this? Would like to upgrade to graphql ^0.9.0 too.

regards
Hannes

**Comment by mickhansen** [comment]
PR has lint errors

**Pull request merged**